src/Resources/skeleton/authenticator/login_form.tpl.php

```diff
@@ -25,6 +25,29 @@
     <input type="hidden" name="_csrf_token"
            value="{{ csrf_token('authenticate') }}"
     >
+<?php if($support_remember_me): ?>
+    <?php if($always_remember_me): ?>
+
+    {#
+        Uncomment this section and comment the remember_me.always_remember_me option below your
+        firewall to let the user to opt-out of the remember me feature
+        See https://symfony.com/doc/current/security/remember_me.html#forcing-the-user-to-opt-out-of-the-remember-me-feature
+
+        <div class="checkbox mb-3">
+            <label>
+                <input type="checkbox" name="_remember_me"> Remember me
+            </label>
+        </div>
+    #}
```

Comment:
I don't think we need this first section at all: if they choose "always remember me", let's not generate anything in this template.